autogluon/utils/tabular/ml/models/rf/rf_model.py

```diff
@@ -124,3 +124,13 @@ def hyperparameter_tune(self, X_train, X_test, Y_train, Y_test, scheduler_option
         hpo_model_performances = {self.name: self.val_score}
         hpo_models = {self.name: self.path}
         return hpo_models, hpo_model_performances, hpo_results
+
+    def get_model_feature_importance(self):
+        if self.features is None:
+            # TODO: Consider making this raise an exception
+            logger.warning('Warning: get_model_feature_importance called when self.features is None!')
+            return dict()
+        feature_names = self.features
+        importances = self.model.feature_importances_
```

Comment:
Will these be on similar scale as those computed by the other models using AbstractModel. compute_permutation_importance()?

Reply:
ditto